app/src/hvr/java/com/igalia/wolvic/PlatformActivity.java

```diff
@@ -95,11 +109,77 @@ private void handlemHmsMessageBroadcast(Intent intent) {
         if (!WolvicHmsMessageService.MESSAGE_RECEIVED_ACTION.equals(intent.getAction()))
             return;
 
-        RemoteMessage.Notification notification = intent.getParcelableExtra(WolvicHmsMessageService.NOTIFICATION_EXTRA);
-        Log.i(TAG, "PushKit received notification: " + notification);
+        RemoteMessage message = (RemoteMessage) intent.getParcelableExtra(WolvicHmsMessageService.MESSAGE_EXTRA);
+        Log.i(TAG, "PushKit: received remote message");
```

Comment:
As a general comment, I think we should use `Log.d` for this kind of messages... Same for the ones bellow

Reply:
Fixed, now all debug messages use `Log.d` except the most relevant ones.